skyportal/handlers/api/analysis.py

```diff
@@ -538,6 +540,16 @@ def post(self):
                     type: float
                     description: Max time in seconds to wait for the analysis service to complete. Default is 3600.0.
                     default: 3600.0
+                  is_summary:
+                    type: boolean
+                    description: |
+                        Established that the results on the resource as a summary
```

Comment:
Establishes the results on the resource as a summary?

Reply:
Done.